heareval/model/baseline.py

```diff
@@ -133,7 +132,7 @@ def frame_audio(
 def get_audio_embedding(
     audio: Tensor,
     model: torch.nn.Module,
-    frame_rate: float,
+    hop_size: float,
```

Comment:
Since discussions to allow hop_size to be defined by participants, let's just assign a default value for this? Say 25ms?

Reply:
Also do we want this value to be in ms as opposed to seconds?